content/docs/0.16.x/manage/shipyard/index.md

```diff
@@ -1,290 +1,103 @@
-
-The action task indicates that a remediation action should be executed by an action provider. It is used within a [remediation workflow](../../automated_operations/remediation).
-
-**Usage:**
-
-```
-- name: action
-```
-
-#### Approval
-
-The approval task intercepts the task sequence and waits for a user to approve or decline the open approval. This task can be added, for example, before deploying an artifact into the next stage. The approval strategy can be defined based on the evaluation result `pass` and `warning`. Keptn supports the following approval strategies for the evaluation results `pass` and `warning`:
-
-* `automatic`: The artifact is deployed automatically.
-* `manual`: The user is asked for approval before triggering the deployment.
-
-This allows combinations as follows:
-
-|                          | Evaluation result: pass           | Evaluation result: warning                 | Behavior  |
-|--------------------------|-----------------------------------|--------------------------------------------|-----------|
-| **Skip approval task:** | pass:automatic | warning:automatic | Regardless of the evaluation result, the approval task is skipped |
-| **Depending on evaluation result:**   | pass:automatic | warning:manual    | If the evaluation result is a **warning**, an approval is required |
-| **Depending on evaluation result:**   | pass:manual    | warning:automatic | If the evaluation result is a **pass**, an approval is required |
-| **Mandatory approval task:**          | pass:manual    | warning:manual    | Regardless of the evaluation result, an approval is required |
-
-By default, an `automatic` approval strategy is used for evaluation result `pass` and `warning`.
-
-**Usage:**
-
-```
-- name: approval
-  properties: 
-    pass: automatic
-    warning: manual
-```
-
-**Note:** If the approval is the first task of a sequence, the approval service is not provided with the result from a previous task. In this case, it always falls back to the `manual` approval strategy.
-
-<details><summary>**Example:** Extended shipyard with a mandatory approval task in production</summary>
-
-<p>
-
-```yaml
-apiVersion: spec.keptn.sh/0.2.3
-kind: "Shipyard"
-metadata:
-  name: "shipyard-sockshop"
-spec:
-  stages:
-    - name: "production"
-      sequences:
-        - name: "delivery"
-          tasks:
+    apiVersion: spec.keptn.sh/0.2.3
+    kind: "Shipyard"
+    metadata:
+      name: "shipyard-sockshop"
+    spec:
+      stages:
+        - name: "dev"
+          sequences:
+          - name: "delivery"
+            tasks: 
             - name: "deployment"
-            - name: "approval"
-              properties:
-                pass: "manual"
-                warning: "manual"
             - name: "release"
-```
-
-</p>
-</details>
-
-#### Deployment
-
-Defines the deployment strategy (see [Continuous Delivery](../../continuous_delivery/)) used to deploy a new version of a service. Keptn supports deployment strategies of type:
```

Comment:
See notes above re: `jmeter-service` and `helm-service`. Drawing the difference between Keptn control plane and the tooling services. The control plane technically doesn't support any deployment strategies. This is old wording (and still technically true because Keptn still ships with the jmeter-service and `helm-service` in the core but (as above) this will soon no longer be the case.

All that said, actually it is the `helm-service` with it's assumption that Istio is installed on the customer cluster that really "supports" these deployment options.

Reply:
Okay, so I do need to do some rewriting here.  And I guess I need to start reflecting the Job Executor Service here now.   Let me do some research.

Reply:
Updated this material in the reference file